frontend/public/components/image-stream.tsx

```diff
@@ -18,6 +18,46 @@ const menuActions = [
   ...common,
 ];
 
+// A valid specTag has a 'builder' tag. no 'hidden' tag, and exists in 'status.tags'.
+export const getBuilderSpecTagName = (imagestream) => {
+  let validSpecTag: any;
+
+  if (!imagestream.status) {
+    return null;
+  }
+
+  if (imagestream.spec && imagestream.spec.tags) {
```

Comment:
You should be able to remove this `if`. `imagestream` will always have a `spec` and `_.find` handles a null `tags`.